.clang-format

```diff
@@ -59,7 +59,7 @@ BreakBeforeInheritanceComma: true
 BreakBeforeTernaryOperators: false
 BreakConstructorInitializersBeforeComma: true
 BreakStringLiterals: true
-ColumnLimit: 80
+ColumnLimit: 100
```

Comment:
Please revert this change. Our policy is to keep the lines not longer than 80 characters.